cloudinit/sources/helpers/vultr.py

```diff
@@ -124,7 +165,13 @@ def generate_network_config(interfaces):
 
     # Prepare additional interfaces, private
     for i in range(1, len(interfaces)):
-        private = generate_interface(interfaces[i])
+        interface = interfaces[i]
+
+        # Skip interfaces set not to be configured
+        if "unconfigured" in interface and interface["unconfigured"]:
```

Comment:
Nit: This could be simplified to `if interface.get("unconfigured"):`

Reply:
TIL. Fixed!